src/build_database.py

```
@@ -8,15 +8,20 @@
+from typing import List, Optional, Dict, Any
 from urllib.parse import urljoin
 
+import pandas as pd
 import requests
 from bs4 import BeautifulSoup
 from tqdm import tqdm
 from urllib3.util.retry import Retry
 from requests.adapters import HTTPAdapter
+from sqlalchemy.exc import IntegrityError
+
+from .db_utils import DatabaseManager
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)